tideways/config.php-profiler.php

```diff
@@ -36,9 +36,43 @@
     ),
     // Other config
     'profiler.enable' => function() {
-        if ( isset( $_SERVER['REQUEST_URI'] ) && strpos( $_SERVER['REQUEST_URI'], 'vvv.test/' ) === 0 ) { //phpcs:ignore
+        $is_vvv_tideways = false;
+
+        if ( empty( $_SERVER['REMOTE_ADDR'] ) and !isset( $_SERVER['HTTP_USER_AGENT'] ) and count( $_SERVER['argv']) > 0 ) {
+            // CLI
+            if ( isset( $_SERVER['ENABLE_TIDEWAYS_CLI'] ) && $_SERVER['ENABLE_TIDEWAYS_CLI'] === '1' ) {
+                $is_vvv_tideways = true;
+            }
+        } else {
+            // Web requests:
+            if ( !isset( $_SERVER['HTTP_HOST'] ) ) {
+                return false;
+            }
+
+            if ( strpos( $_SERVER['HTTP_HOST'], 'vvv.test' ) != false ) {
+                return false;
+            }
+            
+            try { 
+                if ( file_exists( '/srv/config/tideways.json' ) && 
+                    in_array( $_SERVER['HTTP_HOST'], json_decode( file_get_contents( '/srv/config/tideways.json' ) ) ) 
+                   ) {
+                    $is_vvv_tideways = true;
+                }
+            }  
```

Comment:
`json_decode` only throws an error when a particular parameter is used, instead something like this seems better than a try catch:

```php
            if ( file_exists( '/srv/config/tideways.json' ) ) {
                $data = json_decode( file_get_contents( '/srv/config/tideways.json' ) );
                if ( !empty( $data ) && in_array( $_SERVER['HTTP_HOST'], $data ) {
                    $is_vvv_tideways = true;
                }
            }
```

The other side is that if an exception did get thrown it would print out the message at the top of every request